AdyenActions/Actions/RedirectAction.swift

```diff
@@ -25,17 +40,37 @@ public struct RedirectAction: Decodable {
     /// - Parameters:
     ///   - url: The URL to which to redirect the user.
     ///   - paymentData: The server-generated payment data that should be submitted to the `/payments/details` endpoint.
-    ///   - nativeRedirectData: Native redirect data.
+    ///   - type: The redirect flow  used by the action. Defaults to `redirect`.
+    ///   - nativeRedirectData: Native redirect data. Defaults to `nil`.
     ///   - paymentMethodType: The type of the payment method.
     public init(
         url: URL,
         paymentData: String?,
+        type: RedirectType = .redirect,
         nativeRedirectData: String? = nil,
         paymentMethodType: String? = nil
```

Comment:
@erenbesel can you review the changes here? This includes your latest addition of `paymentMethodType`.